src/libFLAC/cpu.c

```diff
@@ -247,7 +245,7 @@ ppc_cpu_info (FLAC__CPUInfo *info)
 #define PPC_FEATURE2_ARCH_2_07		0x80000000
 #endif
 
-#ifdef __linux__
+#if defined (__linux__) && defined(HAVE_GETAUXVAL)
```

Comment:
Shouldn't the first elif that follows this line also get a `&& defined(HAVE_GETAUXVAL)`? Like so:
```
#elif defined(__FreeBSD__) && defined(HAVE_GETAUXVAL)
```

Reply:
I updated the PR as requested